dts/bindings/pwm/nxp,pca9685.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022, Nick Ward <nix.ward@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP PCA9685 16-channel, 12-bit PWM Fm+ I2C-bus LED controller
+
+compatible: "nxp,pca9685"
+
+include: [pwm-controller.yaml, i2c-device.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  outdrv_open_drain:
+    type: boolean
+    description:
+      The 16 LEDn outputs are configured with an open-drain structure.
+      Otherwise they are configured with a totem pole structure.
+
+  och-on-ack:
+    type: boolean
+    description:
+      Outputs change on ACK.  Otherwise the outputs change on STOP
+      command.
+
+  invrt:
```

Comment:
Rename to `invert` to match linux binding

Reply:
Done